legacy/tests/benchmark/test_runner.py

```diff
@@ -0,0 +1,246 @@
+import copy
+import os
+
+import pytest
+
+from .. import _fixtures
+from ..helpers import _uuid
+from ._example_benchmarks import (
+    CasesBenchmark,
+    DataAndErrorBenchmark,
+    ExternalBenchmark,
+    ExternalBenchmarkWithWrongName,
+    SimpleBenchmark,
+    SimpleBenchmarkThatFails,
+    SimpleBenchmarkWithClusterInfo,
+)
+
+
+def validate_benchmark_result(result: dict) -> bool:
+    os.environ["CONBENCH_INTENDED_BASE_URL"] = "http://127.0.0.1:5000/"
+    from conbench.entities.benchmark_result import BenchmarkResultFacadeSchema
+
+    return BenchmarkResultFacadeSchema.create.validate(result)
```

Comment:
Added this; hate it. This is the one place `conbenchlegacy` still depends on the `conbench` package. I'm not sure there's a good way to separate in unit tests here, as copying the file would require copying a significant part of the webapp, and would risk this getting out of sync with the real one anyway, defeating the purpose.

A better solution would be integration tests that use a live Conbench instance to validate the created json is correct.

Open to other ideas.

Reply:
yeah I agree this is the best option for now. We'll set up integration tests eventually. #720